Gemfile.lock

```diff
@@ -288,9 +288,15 @@ GEM
     rb-inotify (0.9.5)
       ffi (>= 0.5.0)
     rdoc (4.2.0)
+<<<<<<< HEAD
```

Comment:
Yes i had a lot of problems with rebase and conflicts)) 
I'll follow your advice